sdk/eventhub/event-hubs/src/linkEntity.ts

```diff
@@ -129,7 +129,7 @@ export class LinkEntity {
     // Although node.js is single threaded, we need a locking mechanism to ensure that a
     // race condition does not happen while creating a shared resource (in this case the
     // cbs session, since we want to have exactly 1 cbs session per connection).
-    log.link(
+    log.info(
```

Comment:
I would expect a log to the `info` channel in the begininng and the end of this function to denote start and end of the claim negotiation process and everything else to be `verbose`